guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -108,8 +108,8 @@
     "        - labels.csv: 'B-Person', 'I-Person', 'I-Person', 'L-Person', 'O', 'O', 'O', 'U-Country'\n",
```

Comment:
<p>Suggestion: Can equally space arguments. Please consider the following:</p><p> </p><p><code>model_folder = os.path.join('data', 'EntityRecognizer', 'models', 'crime_10')</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>